aptos-move/aptos-debugger/src/aptos_debugger.rs

```diff
@@ -185,8 +185,7 @@ impl AptosDebugger {
         version: Version,
     ) -> Result<Option<AnnotatedAccountStateBlob>> {
         let state_view = DebuggerStateView::new(self.debugger.clone(), version);
-        let remote_storage = state_view.as_move_resolver();
-        let annotator = AptosValueAnnotator::new(&remote_storage);
```

Comment:
not sure if it's worth it to have as_value_annotator or smt

Reply:
I am still confused why in the current design we need to provide storage interface to print values nicely.. I guess it is for caching but ends up quite ugly. I ll add as `as_value_annotator` to be similar to how Move resolver is used.